source/scale/high-availability-cluster-based-deployment.rst

```diff
@@ -8,462 +8,414 @@ High availability cluster-based deployment
+    "DataSourceReplicas": [
+      "replica_user:replica_password@tcp(replica1.server)/mattermost?charset=utf8mb4,utf8&readTimeout=30s&writeTimeout=30s",
+      "replica_user:replica_password@tcp(replica2.server)/mattermost?charset=utf8mb4,utf8&readTimeout=30s&writeTimeout=30s"
+    ],
+    "DataSourceSearchReplicas": [],
+    "MaxIdleConns": 20,
+    "MaxOpenConns": 300
+  }
 
 .. note::
 
-  1. File storage is assumed to be shared between all the machines that are using services such as NAS or Amazon S3.
-  2. If ``"DriverName": "local"`` is used then the directory at ``"FileSettings":`` ``"Directory": "./data/"`` is expected to be a NAS location mapped as a local directory, otherwise high availability will not function correctly and may corrupt your file storage.
-  3. If you’re using Amazon S3 or MinIO for file storage then no other configuration is required.
+  - No Hard-Coded IPs: Use cluster-level endpoints for AWS RDS to avoid manual management during failover.
+  - Writes go to the master (or failover replica once promoted).
+  - Reads are distributed among available read replicas by Mattermost.
+ 
+Multi-database setup
+^^^^^^^^^^^^^^^^^^^^
 
-If you’re using the Compliance Reports feature in Mattermost Enterprise, you need to configure the ``"ComplianceSettings":`` ``"Directory": "./data/",`` to share between all machines or the reports will only be available from the System Console on the local Mattermost server.
+1.	Update ``DataSource`` and ``DataSourceReplicas``.
+2.	Apply the new database configuration:
 
-Migrating to NAS or S3 from local storage is beyond the scope of this document.
+  - Reload the Configuration: Navigate to **System Console > Environment > Web Server** and select **Reload Configuration from Disk**.
+  - Recycle Connections: Go to **System Console > Environment > Database** and select **Recycle Database Connections**.
 
-Database configuration
-^^^^^^^^^^^^^^^^^^^^^^
+A brief downtime may occur during reconfiguration, similar to a temporary connection loss for end users.
+ 
+Manage failovers
+^^^^^^^^^^^^^^^^^^
 
-.. tip::
+Manual failover: Update ``DataSource`` to point to a new master, and reload and recycle settings via the System Console as outlined above.
 
-  Specifying configuration setting values using Mattermost environment variables ensure that they always take precedent over any other configuration settings.
+Transparent failover: Transparent failover setups are beyond the scope of this document. 
 
-For an AWS High Availability RDS cluster deployment, point the :ref:`datasource <configure/environment-configuration-settings:data source>` configuration setting to the write/read endpoint at the **cluster** level to benefit from the AWS failover handling. AWS takes care of promoting different database nodes to be the writer node. Mattermost doesn't need to manage this. 
+Automatic failover: use database technologies such as PostgreSQL Clustering or Amazon Aurora.
```

Comment:
These two items seem to come from the transparent failover section in the old document. There's no "automatic failover" there, though. I'm not sure what the correct meaning it, but I don't understand these two items.